oeps/processes/oep-0055-proc-project-maintainers.rst

```diff
@@ -0,0 +1,450 @@
+===========================
+OEP-55: Project Maintainers
+===========================
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-0055 <oep-0055-proc-project-maintainers>`
+   * - Title
+     - Project Maintainers
+   * - Last Modified
+     - 2022-03-16
+   * - Authors
+     - Edward Zarecor <e0d@tcril.org>, Sarina Canelake
+       <sarina@tcril.org>, Xavier Antoviaque <antoviaque>, Olivia
+       Ruiz-Knott <oliviaruizknott>, Ned Batchelder <nedbat>, David
+       Joy <davidjoy>
+   * - Arbiter
+     - Jeremy Bowman <jbowman@2u.com>
+   * - Status
+     - Draft
```

Comment:
Should be Provisional or Accepted, unless you plan to merge a Draft of this.

Reply:
I think that the status update would happen just prior to merging.  I would defer to @jmbowman 's recommendation as to status.

Reply:
My $0.02 would be Provisional but with a concrete timeline / set of objectives for what constitutes "implemented". That might be:

- Set of repos included in Open edX is known and clearly enumerated via some single source of truth
- Maintainers are declared for each of the components of Open edX (and have responded accepting the responsibilities)
- A system is in place for providing maintainers with quant / qual feedback on their progress "maintaining project standards"
- Scrum of scrums is meeting

FYI @e0d 2U is trying to think about pilot repos as part of [our rollout plan](https://github.com/edx/open-source-process-wg/issues/31) as well, so it would be good to coordinate a bit and make sure we have a variety of cases across different orgs.

Reply:
This feels like the right list of priorities in the pilot phase.

Reply:
I made this wiki section to help us document pilot phases & timelines. Could we ensure that documentation is put here? https://openedx.atlassian.net/wiki/spaces/COMM/pages/3426418790/Maintainership+of+Open+edX+Repos